### ISSUE TITLE
Accessibility / language 

### ISSUE THREAD

**Issue by andy-mc-donald** [fac-17/ABCD-week2-project#14]
The lighthouse accessibility test flags up that the `<html>` element doesn't have a `[lang]` attribute:

> "If a page doesn't specify a lang attribute, a screen reader assumes that the page is in the default language that the user chose when setting up the screen reader. If the page isn't actually in the default language, then the screen reader might not announce the page's text correctly."

**Comment by victormasson21** [comment]
Rookie mistake, all sorted, thanks for flagging!
